manifests/crdvalidator/04_webhook.yaml

```diff
@@ -18,6 +18,9 @@ webhooks:
         name: crd-validation-webhook
         path: /validate-crd
         port: 9443
+    objectSelector:
+      matchLabels:
+        core.rukpak.io/owner-kind: BundleInstance
```

Comment:
This seems like it would also close out #259 too? Should we update the PR description and also add a "Closes ..." entry for that issue?

Reply:
It sure does, good catch.